### ISSUE TITLE
Error after updating to v0.6.0

### ISSUE THREAD

**Issue by pozil** [salesforce/sfdx-lwc-jest#96]
### Description
I'm updating e-bikes sample app for Winter '20 ([PR here](https://github.com/trailheadapps/ebikes-lwc/pull/81)) and tests are no longer running after I've updated sfdx-lwc-jest to v0.6.0.

I'm getting dependency errors on the first import of jest tests (full stack in actual results):
> TypeError: Cannot read property 'prototype' of undefined

#### Steps to Reproduce
Run these commands:
```bash
git clone git@github.com:trailheadapps/ebikes-lwc.git
cd ebikes-lwc
git checkout prerelease/winter20
npm install
npm run test:unit
```

#### Expected Results
All tests pass (they were passing prior to sfdx-lwc-jest version update).

#### Actual Results

Tests are failing due to multiple errors like this one:

```
FAIL  force-app/main/default/lwc/productTileList/__tests__/productTileList.test.js
  ● Test suite failed to run

    TypeError: Cannot read property 'prototype' of undefined

    > 1 | import { createElement } from 'lwc';
        | ^
      2 | import ProductTileList from 'c/productTileList';
      3 | import { fireEvent } from 'c/pubsub';
      4 | import {

      at Object.<anonymous> (node_modules/@lwc/engine/src/env/dom.ts:9:68)
      at Object.<anonymous> (node_modules/@lwc/engine/src/shared/assert.ts:10:1)
      at Object.<anonymous> (node_modules/@lwc/engine/src/framework/upgrade.ts:7:1)
      at Object.<anonymous> (node_modules/@lwc/engine/src/framework/main.ts:13:1)
      at Object.<anonymous> (force-app/main/default/lwc/productTileList/__tests__/productTileList.test.js:1:1)
```

### Version
- Salesforce api version 47.0
- @salesforce/sfdx-lwc-jest: 0.6.0
- Node: 10.16.3

**Comment by pozil** [comment]
I can also reproduce on a project created from scratch.

Run these commands:

```bash
sfdx force:project:create -n test-win20
npm init
npm install
npm install @salesforce/sfdx-lwc-jest@prerelease --save-dev
sfdx force:lightning:component:create -n hello --type lwc -d force-app/main/default/lwc
mkdir force-app/main/default/lwc/hello/__tests__
vi force-app/main/default/lwc/hello/__tests__/hello.test.js
```

Paste this content in vim and save the file:
```js
import { createElement } from 'lwc';
import Hello from 'c/hello';

describe('c-hello', () => {

    it('displays nothing', () => {
        // Create element
        const element = createElement('c-hello', {
            is: Hello
        });
        document.body.appendChild(element);

        // Verify that nothing is displayed
        const elements = element.shadowRoot.querySelectorAll('*');
        expect(elements.length).toBe(0);
    });
});
```

Run tests:
```bash
npm run test:unit
```

Output:
```bash
> test-win20@1.0.0 test:unit /Users/pozil/git/web-components/test-win20
> lwc-jest

 FAIL  force-app/main/default/lwc/hello/__tests__/hello.test.js
  ● Test suite failed to run

    TypeError: Cannot read property 'prototype' of undefined

    > 1 | import { createElement } from 'lwc';
        | ^
      2 | import Hello from 'c/hello';
      3 | 
      4 | describe('c-hello', () => {

      at Object.<anonymous> (node_modules/@lwc/engine/src/env/dom.ts:9:68)
      at Object.<anonymous> (node_modules/@lwc/engine/src/shared/assert.ts:10:1)
      at Object.<anonymous> (node_modules/@lwc/engine/src/framework/upgrade.ts:7:1)
      at Object.<anonymous> (node_modules/@lwc/engine/src/framework/main.ts:13:1)
      at Object.<anonymous> (force-app/main/default/lwc/hello/__tests__/hello.test.js:1:1)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        2.128s
Ran all test suites.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! test-win20@1.0.0 test:unit: `lwc-jest`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the test-win20@1.0.0 test:unit script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/pozil/.npm/_logs/2019-10-07T12_57_35_435Z-debug.log
```

**Comment by pozil** [comment]
@trevor-bliss I just tried with 0.6.1-alpha2 and no luck, I still have the same issue.

**Comment by pozil** [comment]
Fixed after moving to 0.6.1.
